src/cli/commands/init.ts

```diff
@@ -198,6 +200,20 @@ async function promptConfigSettings(disablePrompts: boolean, detectedConfig: Fra
       initial: detectedConfig.apiLocation,
       format: trimValue,
     },
+    {
+      type: "text",
```

Comment:
Should be a list with predefined options

Reply:
Added, please check now.